### ISSUE TITLE
[vcf~]

### ISSUE THREAD

**Issue by Tronhjem** [enzienaudio/heavy#249]
[vcf~] works with float input for cutoff frequency in PD but only with signal in Heavy. 
Not a big issue, just an observation.

**Comment by diplojocus** [comment]
@Tronhjem you're right! I actually have a potential fix for that I'm looking to push through. Thanks for pointing it out
